src/web/app/pages-student/student-course-details-page/student-course-details-page.component.ts

```diff
@@ -100,19 +98,24 @@ export class StudentCourseDetailsPageComponent implements OnInit {
       if (!this.teammateProfiles) {
         this.statusMessageService.showWarningMessage('You do not have any teammates yet.');
       }
+
+      if (this.teammateProfiles && this.course && this.student) {
+        const courseId: string = this.course.id;
+        const studentEmail: string = this.student.email;
+        this.teammateProfiles.forEach((teammateProfile: TeammateProfile) => teammateProfile.photoUrl
```

Comment:
The `photoUrl` is always the same?

I guess you should make use of the `/webapi/students` to get the student attributes of the team members.

Reply:
Yup updated